go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/grafana/grafana
 
-go 1.22.3
+go 1.22.4
```

Comment:
Do you have to set the latest version here? It's supposed to be the minimum supported one and I assume it still works just fine on go1.22.3 and even earlier:

* https://go.dev/doc/modules/gomod-ref#go

A good practice seems to be to support the latest three or two major releases (same as go itself).

Reply:
Please don't comment on merged PRs - we usually don't see these comments... Instead, start a discussion or file an issue.

To answer this though, no, we don't _have to_ set the latest version, but we rather _choose_ to. In this case, it ensured that developers were using an up-to-date version.

> A good practice seems to be to support the latest three or two major releases (same as go itself).

For library modules, this is perhaps not a bad idea. However, this repo is not intended to be consumed as a library (and indeed it's currently quite difficult to do so). We also don't generally intend users to compile this themselves, and rather release binary packages for consumption.